tests/suites/3_stateful_iceberg/00_rest/00_0000_create_and_show.sh

```diff
@@ -10,16 +10,14 @@ cat <<EOF | $BENDSQL_CLIENT_CONNECT
 CREATE CATALOG iceberg_ctl
 TYPE=ICEBERG
 CONNECTION=(
-    URL='s3://testbucket/iceberg_ctl/'
-    access_key_id ='minioadmin'
-    secret_access_key ='minioadmin'
-    ENDPOINT_URL='${STORAGE_S3_ENDPOINT_URL}'
+    TYPE='rest'
```

Comment:
is hms ready for test?

add basic read tests?

Reply:
> is hms ready for test?

I plan to implement in following PRs.


> add basic read tests?

I'm still figuring out how to setup a demo table, maybe need spark. I will finish in next PR.